packages/cryptoverif/cryptoverif.2.08/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+synopsis: "CryptoVerif: Cryptographic protocol verifier in the computational model"
+description: """
+CryptoVerif is an automatic protocol prover sound in the computational model. It can prove
+
+  - secrecy;
+  - correspondences, which include in particular authentication;
+  - indistinguishability between two games.
+
+It provides a generic mechanism for specifying the security assumptions on cryptographic primitives, which can handle in particular symmetric encryption, message authentication codes, public-key encryption, signatures, hash functions.
+
+The generated proofs are proofs by sequences of games, as used by cryptographers. These proofs are valid for a number of sessions polynomial in the security parameter, in the presence of an active adversary. CryptoVerif can also evaluate the probability of success of an attack against the protocol as a function of the probability of breaking each cryptographic primitive and of the number of sessions (exact security).
+
+This software is under development; please use it at your own risk. Comments and bug reports welcome. 
+"""
+maintainer: "Bruno Blanchet <Bruno.Blanchet@inria.fr>"
+authors: "Bruno Blanchet <Bruno.Blanchet@inria.fr>, Pierre Boutry <boutry.pierre@gmail.com>, David Cadé <David.Cade@normalesup.org>, Christian Doczkal <christian.doczkal@mpi-sp.org>, Charlie Jacomme <Charlie.Jacomme@inria.fr>, Benjamin Lipp <Benjamin.Lipp@mpi-sp.org>, and Pierre-Yves Strub <pierre-yves@strub.nu>"
+license: "CECILL-B"
+homepage: "https://cryptoverif.inria.fr"
+dev-repo: "git+https://gitlab.inria.fr/bblanche/CryptoVerif.git"
+bug-reports: "Bruno.Blanchet@inria.fr"
+depends: [ "ocaml" { >= "4.08" } "ocamlfind" { post } "cryptokit" { post } "conf-m4" { post } ]
```

Comment:
Why do you need these "post" dependencies?

Reply:
These "post" dependencies are needed when running CryptoVerif (ocamlfind and cryptokit when generating OCaml implementations from CryptoVerif models; m4 for pre-processing some CryptoVerif models). 
However, they are not needed for building CryptoVerif, that is why I put "post".